src/languages/en.ts

```diff
@@ -2756,6 +2756,10 @@ export default {
         invite: 'Invite them',
         nothing: 'Do nothing',
     },
+    actionableReportMentionWhisperOptions: {
+        create: 'Create it',
+        nothing: 'Do nothing',
```

Comment:
Right now the copy of the whisper is `Heads up, #qp doesn't exist yet. Do you want to create it?`. So I think it makes sense to change this to `Yes` and `No` for now.

We probably can use the ones we already have here https://github.com/Expensify/App/blob/main/src/languages/es.ts#L111-L112